sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LookupError.scala

```diff
@@ -54,68 +54,83 @@ object LookupError {
 
 sealed abstract class Reference extends Product with Serializable {
```

Comment:
Now that this isn't only used for error reporting but also for looking up packages in the preprocessor, should we move it out of LookupError.scala?

Reply:
I should revert that as well.